src/graphlog.rs

```diff
@@ -707,37 +663,41 @@ mod tests {
     fn fork_merge_in_central_edge() -> io::Result<()> {
         let mut buffer = vec![];
         let mut graph = AsciiGraphDrawer::new(&mut buffer);
-        graph.add_node(&8, &[Edge::direct(1)], b"o", b"node 8")?;
-        graph.add_node(&7, &[Edge::direct(5)], b"o", b"node 7")?;
+        graph.add_node(&8, &[Edge::direct(1)], "⬤".as_bytes(), b"node 8")?;
+        graph.add_node(&7, &[Edge::direct(5)], "⬤".as_bytes(), b"node 7")?;
         graph.add_node(
             &6,
             &[Edge::direct(2)],
-            b"o",
+            "⬤".as_bytes(),
             b"node 6\nwith\nsome\nmore\nlines",
         )?;
-        graph.add_node(&5, &[Edge::direct(4), Edge::direct(3)], b"o", b"node 5")?;
-        graph.add_node(&4, &[Edge::direct(1)], b"o", b"node 4")?;
-        graph.add_node(&3, &[Edge::direct(1)], b"o", b"node 3")?;
-        graph.add_node(&2, &[Edge::direct(1)], b"o", b"node 2")?;
-        graph.add_node(&1, &[], b"o", b"node 1")?;
+        graph.add_node(
+            &5,
+            &[Edge::direct(4), Edge::direct(3)],
+            "⬤".as_bytes(),
+            b"node 5",
+        )?;
+        graph.add_node(&4, &[Edge::direct(1)], "⬤".as_bytes(), b"node 4")?;
+        graph.add_node(&3, &[Edge::direct(1)], "⬤".as_bytes(), b"node 3")?;
+        graph.add_node(&2, &[Edge::direct(1)], "⬤".as_bytes(), b"node 2")?;
+        graph.add_node(&1, &[], "⬤".as_bytes(), b"node 1")?;
 
         insta::assert_snapshot!(String::from_utf8_lossy(&buffer), @r###"
-        o node 8
-        | o node 7
-        | | o node 6
-        | | | with
-        | | | some
-        | | | more
-        | | | lines
-        | o |   node 5
-        | |\ \  
-        | o | | node 4
-        |/ / /  
-        | o | node 3
-        |/ /  
-        | o node 2
-        |/  
-        o node 1
+        ⬤ node 8
+        │ ⬤ node 7
+        │ │ ⬤ node 6
+        │ │ │ with
+        │ │ │ some
+        │ │ │ more
+        │ │ │ lines
+        │ ⬤─│─╮ node 5
```

Comment:
There can be more crossing. That's the price for straight edges together with a greedy algorithm...